admin/commands/execution/stop_at_height.go

```diff
@@ -0,0 +1,81 @@
+package execution
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/onflow/flow-go/admin"
+	"github.com/onflow/flow-go/admin/commands"
+	"github.com/onflow/flow-go/engine/execution/ingestion"
+)
+
+var _ commands.AdminCommand = (*StopAtHeightCommand)(nil)
+
+// StopAtHeightCommand will send a signal to engine to stop/crash EN
+// at given height
+type StopAtHeightCommand struct {
+	stopAtHeight *ingestion.StopAtHeight
+}
+
+func NewStopAtHeightCommand(sah *ingestion.StopAtHeight) *StopAtHeightCommand {
+	return &StopAtHeightCommand{
+		stopAtHeight: sah,
+	}
+}
+
+type StopAtHeightReq struct {
+	height uint64
+	crash  bool
+}
+
+func (s *StopAtHeightCommand) Handler(ctx context.Context, req *admin.CommandRequest) (interface{}, error) {
+	sah := req.ValidatorData.(StopAtHeightReq)
+
+	_, oldHeight, oldCrash, err := s.stopAtHeight.Set(sah.height, sah.crash)
+
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info().Msgf("admintool: EN will stop at height %d and crash: %t, previous values: %d %t", sah.height, sah.crash, oldHeight, oldCrash)
+
+	return "ok", nil
+}
+
+func (s *StopAtHeightCommand) Validator(req *admin.CommandRequest) error {
```

Comment:
A brief doc would be great. Even an admin command might discover an internal state corruption (like reading a value from the DB that should be present surprisingly errors). Therefore, the default assumption is also here that only error returns explicitly documented as such can be considered benign (-> [guidelines](https://github.com/onflow/flow-go/blob/master/CodingConventions.md#best-practice-guidelines))